eng/scripts/Create-APIView.ps1

```diff
@@ -45,6 +45,27 @@ function Get-AutoRestConfigInfo {
   return $null
 }
 
+function Get-ImpactedTypespecProjects {
```

Comment:
Isn't this logic mostly implemented already in https://github.com/Azure/azure-rest-api-specs/blob/main/eng/scripts/Get-TypeSpec-Folders.ps1?